test/runtests.jl

```diff
@@ -1,6 +1,25 @@
 using GeoStatsPlots
-using Test
+using Variography
+using GeoStatsBase
+using GeoStatsImages
+using CSV
+using Meshes
+using Distributions
+using DelimitedFiles
+using DensityRatioEstimation
+using Plots; gr(size=(600, 400))
+using ReferenceTests, ImageIO
+using Test, Random
+
+# temporary solution
+using GeoStatsPlots: hscatter
```

Comment:
We should export this function, right?

Reply:
We have already exported this function. I did this because both GeoStatsBase and GeoStatsPlots packages export this function.